src/Session.cpp

```diff
@@ -17,36 +26,34 @@ void Session::fillContentFromJson(const std::string &configFilePath)
 
     for(auto movieDesc: contentJson["movies"]) //iterate over movies and add them to content
     {
-        Movie* movieObj = new Movie(currentId, movieDesc["name"], movieDesc["length"], this->extractTags(movieDesc["tags"]));
+        Movie* movieObj = new Movie(currentId, movieDesc["name"],
+                                     movieDesc["length"],
+                                     this->extractTags(movieDesc["tags"]));
```

Comment:
you may put them all in one line. it is either this or that
like this is perfect as well